index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
+<style>
+table, th, td {
+    border: 1px solid black;
+}
+</style>
+</div class="header">
+
+<div id ="nav">
+    <a href="https://ee491f.github.io/course-material/#top">Home</a> |
+    <a href="https://ee491f.github.io/course-material/#html">HTML</a> |
+    <a href="https://ee491f.github.io/course-material/#css">CSS</a> |
+    <a href="https://ee491f.github.io/course-material/#javascript">JavaScript</a> |
+</div id ="nav">
+<hr>
+
+<body>
+
+    <h1>Igloo Savings</h1>
+<table>
+    <tr>
+        <th>Month</th>
+        <th>Savings</th>
+    </tr>
+    <tr>
+        <td>January</td>
+        <td>$100</td>
+    </tr>
+    <tr>
+        <td>February</td>
+        <td>$20</td>
+    </tr>
+    <tr>
+        <td>March</td>
+        <td>$0</td>
+    </tr>
+    <tr>
+        <td>April</td>
+        <td>$0</td>
+    </tr>
+</table>
+<head>
+    <h2>An Igloo</h2>
+    <p>An igloo is made of <span style="color:blue;font-family:Times New Roman">ice</span>.</p>
+    <img src="https://i.pinimg.com/originals/c2/c9/c0/c2c9c0b730a0933a45e0bd120a55fd57.jpg" style="width: 500px; height:600px" alt="igloo.jpg">
+    
+</head>
```

Comment:
The `<head>` tag is usually not nested inside the `<body>` tag
For example:
```
<html>
  <head></head>
  <body></body>
</html>
```
is more typical than:
```
<html>
  <body>
      <head></head>
  </body>
</html>
```